.github/workflows/create-release-branch.yaml

```
@@ -16,3 +16,69 @@ jobs:
     secrets:
       APP_ID: ${{ secrets.APP_ID }}
       AUTOMATION_KEY: ${{ secrets.AUTOMATION_KEY }}
+  
+  create-backport-label:
+    needs: create-release-branch
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Backport PR label if it doesn't exist
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const tagName = context.payload.release?.tag_name;
+            if (!tagName) {
+              console.error("❌ No tag name found in the release payload");
+              throw new Error("No tag name found");
+            }
+            // check if it matches vX.Y.Z format or something/vX.Y.Z
+            const tagPattern = /^(.*\/)?v\d+\.\d+\.\d+$/;
```

I'd suggest being a bit more strict. git tags are already even stricter than the following, with the exception that they will allow more `.` characters. This basically says no`.`s in the prefix

```suggestion
            const tagPattern = /^([\w/-_]*\/)?v\d+\.\d+\.\d+$/;
```